frontend/src/routes/project/[project_code]/+page.svelte

```diff
@@ -40,11 +40,11 @@ import FormatDate from '$lib/components/FormatDate.svelte'
 
 		<div class="flex flex-wrap gap-3">
 			{#each project.ProjectUsers as member}
-				<div class="badge badge-xl badge-primary pr-0">
+				<div class="badge badge-lg badge-primary pr-0">
```

Comment:
I'd start it out in the `routes` area with the narrowest context and either walk it up the routes tree or move it to $lib depending on its pervasiveness across the entire app.

Reply:
ok so what's the narrowest context here? in the `[project_code]` folder? obviously it can move later as needed but I want to understand what you're suggesting I do for the current case where it's not being used anywhere else.

Reply:
yep, I'd put in right next to the page where it's used... if you end with more routes under `[project_code]` and they need it too, it will already be in the right place.  That's just a stylistic option, if you prefer putting under `$lib` it would be ok too.  I just find these custom comps better close to where they're needed if they aren't generic to the whole app.